flocker/provision/_install.py

```diff
@@ -1250,6 +1280,7 @@ def maybe_disable(unit):
 _flocker_uninstallers = {
     "ubuntu-14.04": _uninstall_flocker_ubuntu1404,
     "centos-7": _uninstall_flocker_centos7,
+    "rhel-7.2": _uninstall_flocker_centos7,
```

Comment:
15.10 support is still partial (https://docs.clusterhq.com/en/latest/supported/index.html#supported-operating-systems does not list 15.10, and we do not have tests running on 15.10 except for benchmark work). Thanks for pointing out. @wallrj is planning to work on 16.10 support - we will update the thread with 15.10 plan soon.